ui/pages/permissions-connect/snaps/snap-install/snap-install.js

```diff
@@ -219,7 +226,11 @@ export default function SnapInstall({
           cancelButtonType="default"
           hideCancel={hasError}
           disabled={
-            isLoading || (!hasError && isScrollable && !isScrolledToBottom)
+            isLoading ||
+            (!hasError &&
+              isScrollable &&
+              !hasScrolledToBottom &&
+              !showAllPermissions)
```

Comment:
I'm having trouble reasoning about why `showAllPermissions` needs to be part of this condition. Can you explain?

Reply:
I think this was left from the moment when `useScrollRequired` was not working properly, until it's fixed by Guillaume. I think this is redundant now, so I removed it ☑️ .